module/importer-magento-1/src/Infrastructure/Reader/Magento1CsvReader.php

```diff
@@ -10,11 +10,10 @@
 
```

Comment:
I'd opt-in for having unit tests for this

Reply:
In my opinion, u can do it simplier with: https://github.com/thephpleague/csv?

Reply:
Reader class will be refactor in next step